router/delivery/http/router_handler.go

```diff
@@ -136,21 +144,24 @@ func (a *RouterHandler) GetOptimalQuote(c echo.Context) (err error) {
 	return c.JSON(http.StatusOK, quote)
 }
 
-// GetDirectCustomQuote returns a direct custom quote. It does not search for the route.
-// It directly computes the quote for the given poolID.
+// @Summary Compute the quote for the given poolID
+// @Description Call does not search for the route rather directly computes the quote for the given poolID.
+// @ID get-direct-quote
+// @Produce  json
+// @Param  tokenIn         query  string  true  "String representation of the sdk.Coin for the token in."                   example(5OSMO)
+// @Param  tokenOutDenom   query  string  true  "String representing the list of the token denom out separated by comma."   example(ATOM,USDC)
+// @Param  poolID          query  string  true  "String representing list of the pool ID."                                  example(1,2,3)
+// @Param  applyExponents  query  bool    false  "Boolean flag indicating whether to apply exponents to the spot price. False by default."
+// @Success 200  {object}  domain.Quote  "The computed best route quote"
+// @Router /router/custom-direct-quote [get]
 func (a *RouterHandler) GetDirectCustomQuote(c echo.Context) error {
```

Comment:
We should cover this endpoint by our new integration test suite.

Please see reference here (similar to `/router/quote`:
https://github.com/osmosis-labs/sqs/blob/d63e8c937378c68704976de86cf2d59cdfd6a0e0/tests/test_router_quote.py#L31

Please see this for set up:
https://github.com/osmosis-labs/sqs/blob/d63e8c937378c68704976de86cf2d59cdfd6a0e0/Makefile#L190-L202